content/applications/hr/recruitment/recruitment_analysis.rst

```diff
@@ -0,0 +1,103 @@
+----------------
+
+For a more detailed view of the information in the :guilabel:`Recruitment Analysis` report, click
+the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. This displays all the
+information in a pivot table.
+
+In this view, the job positions are displayed in the rows, and the columns display the total numbers
+of applicants, including how many of those applicants were hired or refused. The displayed
+information can be modified, if desired.
+
+In this example, there are 17 total applicants. Out of that 17, three have been hired, and four
+refused. Each job position lists the corresponding number of applicants hired and refused. The
+:guilabel:`Experienced Developer` position has eight total applicants, two of which were hired, and
+two were refused.
+
+.. image:: recruitment_analysis/pivot-view.png
+   :align: center
+   :alt: The detailed pivot table view.
+
+Use case: applicants with referrals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get a better understanding of how effective the company's :doc:`referral program <../referrals>`
+is, the :guilabel:`Recruitment Analysis` report can be modified to show how many applicants were
+referred by current employees.
+
+From the :icon:`oi-view-pivot` :guilabel:`(Pivot)` view of the :guilabel:`Recruitment Analysis`
+report, first click on the :guilabel:`Measures` drop-down button.
+
+Click both :guilabel:`Has Referrer` and :guilabel:`Count`, to activate those two measures. Then,
+click :guilabel:`# Applicant`, :guilabel:`# Hired`, and :guilabel:`# Refused` to deactivate those
+default measures.
+
+Now, the column displays the number of applicants that came from a referral in the :guilabel:`Has
+Referral` column, and the total number of applicants in the :guilabel:`Count` column.
+
+.. image:: recruitment_analysis/referral.png
+   :align: center
+   :alt: The detailed pivot table view displaying the number of referrals and the total applicants.
+
+In this example, the :guilabel:`Experienced Developer` job position has the most applicants from
+referrals. Out of the eight applicants, six have applied through a referral from a current employee.
+Meanwhile, the :guilabel:`Marketing and Community Manager` job position has the least amount of
+referrals out of the total applicants, only one out of six.
+
+Hired through referrals
+***********************
+
+It is possible to modify this report even further to see how many referred applicants end up being
+hired.
+
+To view this data, click on a :icon:`fa-plus-square` :guilabel:`(Job Position)` row, which reveals a
+drop-down menu. Then, click :guilabel:`State` to show the various states that applicants are
+currently in.
+
+.. note::
+   Only states that have applicants in them are shown for each job position. If a state does **not**
+   have any applicants, it does not appear in the list.
+
+To expand the other rows and display the various states, click on the :icon:`fa-plus-square`
+:guilabel:`(Job Position)` button.
```

Comment:
```suggestion
:guilabel:`(Job)` button.
```
When I hover over that +, the tooltip reads as just "Job" -- so it might be worth double-checking on this during your revisions. Thanks!

Reply:
The text that is pulled is called the 'Job Position" on the main recruitment dashboard. So you see that wording- and what is listed there is what is shown on the report. So even though the tool tip just says "job" I think to be consistent with the rest of the documentation/terminology, I kept them the same.